kuma/wiki/tests/test_templates.py

```diff
@@ -680,6 +680,7 @@ def test_tag_list(self):
         doc = pq(response.content)
         eq_(1, len(doc('#document-list ul.document-list li')))
 
+    @pytest.mark.xfail(reason='Requires migrations')
```

Comment:
I inspected the database in the test by adding a ``pdb`` breakpoint and running this code, in normal test mode and with ``--nomigrations``:

```
from django.db import connection

cursor = connection.cursor()
cursor.execute("SHOW CREATE TABLE wiki_documenttag")
table_name, create_table = cursor.fetchone()
cursor.execute("SELECT @@character_set_database, @@collation_database")
charset, collation = cursor.fetchone()
cursor.close()

print(table_name)
print(create_table)
print(charset)
print(collation)
```

In both cases, the table was ``wiki_documenttag``, the default character set was ``utf8``, and the default collation was ``utf8_general_ci``, not the custom collation of ``utf8_distinct_ci``.

The create statement for a normal run is:
```
CREATE TABLE `wiki_documenttag` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(100) CHARACTER SET utf8 COLLATE utf8_distinct_ci DEFAULT NULL,
  `slug` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `slug` (`slug`),
  UNIQUE KEY `name` (`name`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8
```

The create table for a ``--nomigrations`` run is:
```
CREATE TABLE `wiki_documenttag` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(100) NOT NULL,
  `slug` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `name` (`name`),
  UNIQUE KEY `slug` (`slug`)
) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=utf8
```

The only thing they have in common is the ``varchar(100)``:
```
  `name` varchar(100) CHARACTER SET utf8 COLLATE utf8_distinct_ci DEFAULT NULL,
  `name` varchar(100) NOT NULL,
```

In docker and production, the default character set is ``utf8`` and the collation ``utf8_general_ci``.

In Docker, the the create table is the same as the normal test run:
```
CREATE TABLE `wiki_documenttag` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(100) CHARACTER SET utf8 COLLATE utf8_distinct_ci DEFAULT NULL,
  `slug` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `slug` (`slug`),
  UNIQUE KEY `name` (`name`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

In production, the create table is similar, but is missing the unique key:
```
CREATE TABLE `wiki_documenttag` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(100) CHARACTER SET utf8 COLLATE utf8_distinct_ci DEFAULT NULL,
  `slug` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `slug` (`slug`)
) ENGINE=InnoDB AUTO_INCREMENT=646911 DEFAULT CHARSET=utf8
```

So, ``--nomigrations`` is the odd one out.  Searching for bugs in Django, py.test...